sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -114,13 +116,12 @@ def _dependency_to_requirements(dependency=[], filename='requirements.txt'):
     dependency_helper.add_python_package(version)
   dependency_helper.generate_pip_requirements(filename)
 
-def _generate_dockerfile(filename, base_image, entrypoint_filename, python_version, requirement_filename=None):
+def _generate_dockerfile(filename, base_image, python_version, requirement_filename=None, add_files=None):
   """
     generates dockerfiles
     Args:
       filename (str): target file name for the dockerfile.
       base_image (str): the base image name.
-      entrypoint_filename (str): the path of the entrypoint source file that is copied to the docker image.
       python_version (str): choose python2 or python3
       requirement_filename (str): requirement file name
```

Comment:
Could you add the argument introduction for the add_files?

Reply:
Done